lib/rgit/cli.rb

```diff
@@ -1,31 +1,234 @@
-
+require 'byebug'
+# stdlib imports
+require "pathname"
+# gem imports
 require "thor"
+require "rugged"
+require "parseconfig"
+# Local imports
+require_relative "descriptions"
+require_relative "version"
+# Monkey Patches - import
+require "core_extensions/pathname/easychildcheck.rb"
+# Monkey Patches - apply
+Pathname.include CoreExtensions::Pathname::EasyChildCheck
 
-module Rgit
-  class Cli < Thor
+module Impl
+
+end
+
+module Wrappers
+  class Branches
+    include Impl
+    attr_accessor :current_group, :current_branch
+    def initialize(dir)
+      @groups = get_branches(dir)
+      @dir = dir
+      update
+      @groups2branch = flat_hash(@groups).each.select { |k, v|
+
+      }
+
+    end
+
+    def update
+      @groups = self.get_branches(@dir)
+      @current_branch = git_command("symbolic-ref", "HEAD", @dir)
+                            .strip.gsub("refs/heads/", "")
+      @current_group = @current_branch[1..-1].split("/", 2)[0]
+    end
+
+    def flat_hash(h,f=[],g={})
```

Comment:
Space missing after comma.<br>Surrounding space missing in default value assignment.